src/pcre.js

```diff
@@ -1,3 +1,5 @@
+/* global CodeMirror */
```

Comment:
Can you move this to a separate commit and add an extra comment reflecting this is intended to please JSHint?

Reply:
Sorry for asking: 
- should i delete the line, make a commit, add it again and make a new comment?
- or should i create a new branch that covers the fixes as a commit and the JSHint as a second commit?

Also i have an additional branch, that holds a dependency injection to make it compatible to Laravel/Vue.js, that is based on the current branch ...

Reply:
I would advise to delete the line, amend your commit, and add a second commit with:

- `/* global CodeMirror */`
- a comment in the code reflecting that line is intended for JSHint (that is not very important but that is not obvious either)
- a proper commit message

...then you can `git push --force` (Github handles that quite well).

However, if you have an additional branch of yours, you will likely need to rebase it on top of the second commit, whatever its branch.

Reply:
ok, done

Reply:
ok, next try :)